### PR TITLE
Add num_processes parameter to italian importer for Communication task type

### PR DESCRIPTION
**Pull request by Guilucand**
This change is [Reviewable](https://reviewable.io/reviews/cms-dev/cms/607)

### CODE REVIEW

File: cmscontrib/loaders/italy_yaml.py [hunk 1]
```diff
@@ -583,8 +584,12 @@ def get_task(self, get_statement=True):
                      os.path.join(self.path, "cor", "manager")]
             for path in paths:
                 if os.path.exists(path):
+                    num_processes = load(conf, None, "num_processes")
+                    if (num_processes == None):
```
Comment (stefano-maggiolo) [inline cmscontrib/loaders/italy_yaml.py]: Codacy Issue found: [Unnecessary parens after 'if' keyword](https://www.codacy.com/app/s-maggiolo/cms/file/2485883075/issues/source?bid=3167167&fileBranchId=3265937#l588)

**Comment by wil93** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[cmscontrib/loaders/italy_yaml.py, line 588 \[r1\]](https://reviewable.io:443/reviews/cms-dev/cms/607#-KGTlw_2J2d-Se2HhwAl-r1-588:-KGTtvTFXS6_poCHZenr:-388383244) ([raw file](https://github.com/cms-dev/cms/blob/bb65748b30e9821918d7e8e7297d4c5a00e25f2a/cmscontrib/loaders/italy_yaml.py#L588)):*
Furthermore, it's generally recommended to do `is None` instead of `== None` (http://stackoverflow.com/questions/3257919/is-none-vs-none)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/607)*

**Comment by wil93** [comment]
Reviewed 2 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/607)*

**Comment by stefano-maggiolo** [comment]
Merged!

**Pull request closed without merging**